src/app/comments/comments.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
```suggestion
import {
	Component,
	OnInit
} from '@angular/core';
```
Kiểm tra những chỗ khác. Nếu import từ 2 cái trở lên trong cùng 1 hàng thì  nên xuống dòng.